python/tests/backends/test_IQM.py

```diff
@@ -98,8 +98,6 @@ def test_iqm_ghz():
     assert assert_close(counts["11"], shots / 2, 2)
 
 
-# FIXME: This test relies on the mock server to return the correct
```

Comment:
Is this actually fixed? If not, an issue should be filed.

Reply:
Yes, I believe this is fixed and was just a now inaccurate comment left over.